docs/specs/devcontainer-reference.md

```diff
@@ -81,13 +81,19 @@ Mounts allow containers to have access to the underlying machine, share data bet
 
 It is important to note that these mounts are from the underlying compute environment and thus cloud environments might not have access to the same data as a local machine.
 
+## Workspace folder
+
+The `workspace-folder` is used for the purpose of identifying the path where the configuration files are found. This path is also automatically included in the mounted folders for the container. This mount is by default created pointing to `/workspace` but can be modified with the [`workspaceMount` and `workspaceFolder`](devcontainerjson-reference.md#image-or-dockerfile-specific-properties) properties.
```

Comment:
```suggestion
The `workspaceFolder` is used for the purpose of identifying the path where the configuration files are found. This path is also automatically included in the mounted folders for the container. This mount is by default created pointing to `/workspace` but can be modified with the [`workspaceMount` and `workspaceFolder`](devcontainerjson-reference.md#image-or-dockerfile-specific-properties) properties.
```

Reply:
I think we'd want to explain the relationship of the mount point and the workspace folder a bit more particularly for Remote - Containers.

The workspaceMount should point at to the root of a repository to ensure that git operations work, even if you are actually trying to open a sub-folder in the repository.  You can then set workspaceFolder to a sub-folder under the mount point in the container if you want to open a sub-folder by default.

I think it's also worth pointing out that the `workspaceMount` property is only used in the orchestrator-less scenario since orchestrators have their own ways to mount content (e.g. Docker Compose).

Reply:
Applied this comments.